far/stddlg.hpp

```diff
@@ -54,8 +55,14 @@ enum class lng : int;
 class regex_exception;
 struct error_state_ex;
 
-struct SearchReplaceDlgOptions
+struct SearchReplaceDlgParams
 {
+	bool ReplaceMode{};
+	bool HideButtonAll{};
```

Comment:
This is used only in Editor, in all other places it has to be hidden explicitly, maybe make it true by default, or rather invert as `ShowButtonAll = false`?

Reply:
Invert "Button All" flag only, right? "Replace Mode" is fine? I thought about it, all name variants I thought of are ambiguous in the sense that it is unclear whether we are talking about a particular button, or about all of something. After spending rather much time on these considerations, I decided to keep original meaning at least. I will happily flip it to make default init more natural.

Reply:
Thanks